### ISSUE TITLE
Make a custom $search command

### ISSUE THREAD

**Issue by ritofolf-dev** [SpectrixOfficial/Spectrum#14]
Syntax: $search (search engine) (term)

**Comment by notBanii** [comment]
google has an api, im pretty sure yahoo does too. i dont see why you can't do that.

**Comment by SpectrixOfficial** [comment]
I can do that, it's just that he was very unclear

**Comment by TheLordofGaming** [comment]
I used angle brackets which happened to be HTML format and git didn't like it.

**Comment by SpectrixOfficial** [comment]
So it will be multiple search engines? Ok, that probably isn't that hard...

**Comment by TheLordofGaming** [comment]
It could work with Auxiliatrium. How do you pronounce that, lol?

**Comment by SpectrixOfficial** [comment]
_It's unpronounceable_

**Comment by SpectrixOfficial** [comment]
Also, Auxiliatrium doesn't use the Google API. So I'd have to do something different

**Comment by TheLordofGaming** [comment]
Au-xilia-trium

**Comment by TheLordofGaming** [comment]
:P

**Comment by LevitatingBusinessMan** [comment]
Google really doesn't like scrapers.
I don't recommend this.
If you do use it just use a pip package

**Comment by notBanii** [comment]
Bruv, Google has a search API lol. Just search for it lmao

**Comment by LevitatingBusinessMan** [comment]
@notBanii Yes but for a price... Let's see if you can afford it

**Comment by EnterNewName** [comment]
am i too late to the party :)

**Comment by notBanii** [comment]
@LevitatingBusinessMan https://developers.google.com/custom-search/docs/tutorial/introduction

**Comment by UbnGithub** [comment]
@notBanii That's for websites that want a search bar for content within the site...

**Comment by EnterNewName** [comment]
lmao

**Comment by notBanii** [comment]
Oof

**Comment by SpectrixOfficial** [comment]
Yeah, I'll use a pip package, maybe something [like this](https://github.com/abenassi/Google-Search-API)
